src/ui/interface.js

```diff
@@ -228,4 +230,67 @@ module.exports = function Interface(options) {
     $("[rel=popover]").popover()
     return true;
   });
+
+  //Start Handle multiple webcam resolutions
+  function changeResolution(w, h){
+    document.getElementById('image').setAttribute("width",w);
+    document.getElementById('image').setAttribute("height",h);
+  }
+  $('#qvga').click(function(e){
+    changeResolution('100px','100px')
+  });
+  $('#vga').click(function(e){
+    changeResolution('800px','600px')
+  });
+  $('#hd').click(function(e){
+    changeResolution('2400px','1800px')
+  })
+  $('#full-hd').click(function(e){
+    changeResolution('6000px','4000px')
+  });
+  //End Handling of Multiple webcam resolutions
+
+  //Start Canvas Recording and Download
+
+  const canvas = document.getElementById('image');
+  const ctx = canvas.getContext('2d');
+  var x = 0;
+  const stream = canvas.captureStream(); // grab our canvas MediaStream
+  const rec = new MediaRecorder(stream); 
+
+  function exportVid(blob) {
```

Comment:
This looks great. It's convenient for this all to be near the interface code, but do you imagine a possible future reorganization where it could live in... i'm not sure, the /src/io/ folder? Somewhere else? Since it's not technically part of the interface. We don't have to think about this yet, but we could put a comment in here to eventually move it, or open an issue to think about this in the future.